protocol/x/clob/keeper/process_operations.go

```diff
@@ -551,22 +550,6 @@ func (k Keeper) PersistMatchLiquidationToState(
 		return err
 	}
 
-	notionalQuoteQuantums, err := k.perpetualsKeeper.GetNetNotional(
-		ctx,
-		matchLiquidation.PerpetualId,
-		new(big.Int).SetUint64(matchLiquidation.TotalSize),
-	)
-	if err != nil {
-		return err
-	}
-	absNotionalQuoteQuantums := new(big.Int).Abs(notionalQuoteQuantums)
-
-	telemetry.IncrCounterWithLabels(
-		[]string{types.ModuleName, metrics.LiquidationOrderNotionalQuoteQuantums, metrics.DeliverTx},
-		metrics.GetMetricValueFromBigInt(absNotionalQuoteQuantums),
-		matchLiquidation.GetMetricLabels(),
-	)
-
```

Comment:
this is duplicated - there is a similar metric in process single match that keeps track of filed quote quantums